apps/site/assets/css/_footer.scss

```diff
@@ -219,6 +219,10 @@ address {
         border-radius: 0;
         margin-bottom: 2px;
         margin-top: 0;
+
+        @include media-breakpoint-up(lg) {
```

Comment:
This sorta feels like it defeats the purpose of the `.mobile` class name. Do we know why that class is there? Would it make more sense to remove it from the footer?

Reply:
This class is used in the footer(as well as header) so to change the styles just for footer desktop had to add this breakpoint. Might be worth separating out the footer styles but there would be a lot of duplicate css 💭